### ISSUE TITLE
Profile related error message pops up when no Titanfall2 install is found

### ISSUE THREAD

**Issue by GeckoEidechse** [R2NorthstarTools/FlightCore#499]
Of course I only think about testing without Titanfall2 folder set after merging #494  (/.-)

![image](https://github.com/R2NorthstarTools/FlightCore/assets/40122905/5cde8c4b-3405-4cfd-9fac-0811222878cb)

**Comment by GeckoEidechse** [comment]
cc @Jan200101

**Comment by GeckoEidechse** [comment]
Actually, in further testing, after selecting the Titanfall2 dir manually, the default profile is not set and still stuck on `'''`, which means it looks like


![image](https://github.com/R2NorthstarTools/FlightCore/assets/40122905/a216b081-ba47-4aa6-a0e5-e9da985560f1)
![image](https://github.com/R2NorthstarTools/FlightCore/assets/40122905/3c2b8815-8587-46d5-a92d-1b1db078385d)

even though Northstar is already installed in this case

Settings:

![image](https://github.com/R2NorthstarTools/FlightCore/assets/40122905/f25f002b-8f86-4877-be23-6731ccf1ef3b)


So basically, after selecting install manually, profile list needs to be refreshed and default profile set if none is set.
